test/config_cat_test.exs

```diff
@@ -36,7 +36,8 @@ defmodule ConfigCatTest do
       {:ok, client} = start_config_cat(sdk_key, fetch_policy: FetchPolicy.manual())
 
       APIMock
-      |> stub(:get, fn ^url, _headers ->
+      |> stub(:get, fn ^url, _headers, options ->
+        assert options == []
```

Comment:
#minor This can be stated more concisely as `stub(:get, fn ^url, _headers, [] -> `. I normally don't like to put `assert`s in `stub`s, but was forced to do it when testing the headers.

Reply:
Done 👍